.github/workflows/postman.yml

```
@@ -0,0 +1,70 @@
+          sed -i 's/0279be667ef9dcbbac55a06295ce870b07029bfcdb2dce28d959f2815b16f81798/"{{public_key_hex}}"/g' gateway-api-spec.yaml
+      - name: Update Postman's Gateway API collection from the spec
+        run: |
+          npx @apideck/portman -l gateway-api-spec.yaml --syncPostman true -o tmp.collection.json --envFile .env --cliOptionsFile cliopts.json \
+            --postmanUid 14449947-b6b059b1-67db-46ea-901b-cb446c3c3a1e
+      - name: Download the collection
+        run: |
+          curl -X GET -H "X-API-KEY:${{secrets.POSTMAN_API_TOKEN}}" https://api.getpostman.com/collections/14449947-b6b059b1-67db-46ea-901b-cb446c3c3a1e > tmp.collection.json
+      - name: Add a timestamp to the name
+        run: |
+          sed -i 's/"name":".*","description":"This API/"name":"Gateway API (develop) ${{steps.date.outputs.date}} UTC","description":"This API/' tmp.collection.json
+      - name: Change the baseUrl variable name
+        run: |
+          sed -i 's/{{baseUrl}}/{{gatewayBaseUrl}}/g' tmp.collection.json
+      - name: Update the collection
+        run: |
+          curl -X PUT -H "X-API-KEY:${{secrets.POSTMAN_API_TOKEN}}" -H "Content-Type: application/json" \
+            https://api.getpostman.com/collections/14449947-b6b059b1-67db-46ea-901b-cb446c3c3a1e --data "@tmp.collection.json"
+      - name: Validate the collection against RCnet
+        run: |
+          newman run tmp.collection.json \
+            --environment https://api.getpostman.com/environments/3671706-92569d6f-4d81-4b15-a5a2-d65afc8fdc26?apikey=${{secrets.POSTMAN_API_TOKEN}} \
+            --folder Status --folder Account --folder Token --folder Validator --bail
```

What does the --bail parameter do out of interest?

---

`--bail` tells Newman to fail fast, like it will immediately stop the test runner if a single test fails. 

Actually I don't think it makes sense to bail here. I'll remove it